src/server/scripts/Custom/SkipDK.cpp

```diff
@@ -340,6 +155,43 @@ class Trinitycore_optional_deathknight_skip : public CreatureScript
     }
 };
 
+void Trinitycore_skip_deathknight_HandleSkip(Player* player)
+{
+    //Not sure where DKs were supposed to pick this up from, leaving as the one manual add
+    player->AddItem(6948, true); //Hearthstone
+
+    // these are all the starter quests that award talent points, quest items, or spells
+    int STARTER_QUESTS[39] = { 12593, 12619, 12842, 12848, 12636, 12641, 12657, 12678, 12679, 12680, 12687, 12698, 12701, 12706, 12716, 12719, 12720, 12722, 12724, 12725, 12727, 12733, 12739, 12742, 12743, 12744, 12745, 12746, 12747, 12748, 12749, 12750, 12751, 12754, 12755, 12756, 12757, 12779, 12801 };
```

Comment:
talent points are give when a player levels up, this would ensure accurate. 
quest items depends on if it is rewarded gears, spells are already given with the exsisting module with their appropiate quests being rewarded.
Current module ensures gear and select spells and conditions are met to ensure propper phasing of the ebon hold being phased.

Reply:
Normally talent points are awarded as you level, but DK quests specifically have talent points assigned to them.  There were not any uses of the full character load that you get from restarting the client in code that I could find, so I thought it more correct to actually complete the quests and let the counter increase "naturally" for DK talent points.
The neck piece was missing as a manually added reward.  Other items were correct.  The phasing of ebon hold looked correct when I went back to check on runeforging.  Some of the awarded spells from your implementation were from higher levels of runeforging (level 72 runes) and the Skill 960 does not exist in 3.3.5a.  Was that from retail?  Either way, I added all the quests from 3.3.5 so the manual adds of items or spells were not required.